docs/source/getting-started/setting-up-driver-plugin.adoc

```diff
@@ -140,9 +135,9 @@ $ sudo chmod u+s /usr/local/bin/docker-machine-driver-xhyve
 
 [NOTE]
 ====
-The downloaded `docker-machine-driver-xhyve` binaries are compiled against a specific
-version of OS X. It is possible that the driver fails to work after an OS X version
-upgrade. In this case you can try to compile the driver from source:
+The downloaded *docker-machine-driver-xhyve* binaries are compiled against a specific version of OS X.
```

Comment:
Ok, let's align also how we refer to OS X. According to the latest, it should be macOS. I believe we might use it once or twice, but as we see here, not across the board. Let's do a find&replace of OS X and replace it with macOS.